demo/bottom_up_pose_tracking_demo.py

```diff
@@ -28,26 +28,14 @@ def main():
     parser.add_argument(
         '--kpt-thr', type=float, default=0.5, help='Keypoint score threshold')
     parser.add_argument(
-        '--nms-oks-thr', type=float, default=0.9, help='NMS oks threshold')
+        '--pose-nms-thr', type=float, default=0.9, help='Pose nms threshold')
```

Comment:
I suggest we make it clear that pose-nms-thr is applied on OKS score in the help message.